tel2puml/otel_to_pv/otel_to_pv.py

```diff
@@ -73,11 +80,40 @@ def otel_to_pv(
                     async_event_groups.get(job_name, None)
                 ),
                 event_types_map_information=(
-                    event_name_map_information.get(
-                        job_name, None
-                    )
+                    event_name_map_information.get(job_name, None)
                 ),
             ),
         )
         for job_name, job_id_streams in job_name_group_streams
     )
+    if save_events:
```

Comment:
so didn't see any tests for these error cases in the tests. Would suggest putting this functionality into two functions. One that takes in input job name, pv_event_streams and output file directory and then another that this will call that takes in a pv_event, output_directory